tmva/pymva/src/MethodPyKeras.cxx

```diff
@@ -212,21 +294,64 @@ void MethodPyKeras::ProcessOptions() {
          PyRunString("tf.compat.v1.keras.backend.set_session(sess)");
       }
    }
+   // case not using a Tensorflow backend
    else {
+      fKerasString = "keras";
       if (fNumThreads > 0)
-         Log() << kWARNING << "Cannot set the given " << fNumThreads << " threads when not using tensorflow as  backend"  << Endl;
-      if (!fGpuOptions.IsNull() ) {
-         Log() << kWARNING << "Cannot set the given GPU option " << fGpuOptions << " when not using tensorflow as  backend"  << Endl;
+         Log() << kWARNING << "Cannot set the given " << fNumThreads << " threads when not using tensorflow as  backend"
+               << Endl;
+      if (!fGpuOptions.IsNull()) {
+         Log() << kWARNING << "Cannot set the given GPU option " << fGpuOptions
```

Comment:
Should this be a warning or an error?

Reply:
I think a warning is fine. This has been the case so far